debianpkg/backports/ubuntu14.04/debian/rules

```diff
@@ -137,16 +137,13 @@ override_dh_auto_configure:
 override_dh_auto_build:
 	#dh_auto_build
 	$(MAKE)
-	dh_auto_build -- -C doc draft-zebra-00.txt
-
 
 	# doc/ is a bit crazy
 ifeq ($(GENERATE_PDF), 1)
-	dh_auto_build -- -C doc frr.pdf || true # pdfetex fails with exit code 1 but still produces a good looking .pdf
+	dh_auto_build -- -C doc user.pdf
 endif
-	rm -vf doc/frr.info
-	dh_auto_build -- -C doc frr.info
-	rm -vf doc/frr.info.html*
+	rm -vf doc/_build/texinfo/FRR.info
+	dh_auto_build -- -C doc user.info
```

Comment:
misspelling? Should be `FRR.info` - not `user.info`
dh_auto_build -- -C doc FRR.info